main.go

```diff
@@ -6,27 +6,35 @@ package main
 import (
 	"context"
 	"flag"
+	"fmt"
 	"net"
 	"net/http"
 	"os"
 	"os/signal"
 	"runtime"
+	"runtime/debug"
 	"runtime/pprof"
 	"syscall"
+	_ "unsafe"
 
 	"github.com/prometheus/client_golang/prometheus/promhttp"
 	"github.com/rs/zerolog"
 
 	"github.com/corazawaf/coraza-spoa/internal"
 )
 
+var (
+	Version = "dev"
```

Comment:
```suggestion
	version = "dev"
```

Doesn't need to be exported.